### ISSUE TITLE
Avoid optional HTML?

### ISSUE THREAD

**Issue by j9t** [w3c/sustyweb#40]
The HTML section features some pointers on markup optimization, but misses perhaps the biggest, namely optional features of HTML, whose omission can significantly reduce HTML payload.

These features include [optional tags, quotes, as well as attribute value defaults](https://meiert.com/en/blog/optional-html/) (disclosure: referencing one of my own articles as I’m not aware of a more comprehensive overview).

(This could warrant 1–3 recommendations in the guidelines, depending on how you assess and want to cut them.)

**Comment by AlexDawsonUK** [comment]
Agreed, this would make a good success criteria under "3.8 Use HTML Elements Correctly" to get people pushing for a reductionist approach within their semantics. I'll get something put together for the upcoming draft.

**Comment by AlexDawsonUK** [comment]
This has now been added into the second draft. It will become visible once the document goes public.

**Comment by j9t** [comment]
Awesome! Is there a preview, or some other opportunity to review and give feedback?

**Comment by AlexDawsonUK** [comment]
The second draft itself will be arriving this month so naturally once it's published if you spot any further issues, as always feel free to drop an issue onto the system and it'll be rolled into the third draft in turn.

Regarding the success criteria I've added - I've worded it as the below (and included links within the Resources section).

> Success Criterion - Optional Features
> Consider removing optional HTML tags (which aren't required for rendering), attribute quotes, or attributes that are set to their default value.

I've tried to keep it brief (and let people click through as always if they wish to discover more regarding implementation). If you feel it could be adjusted further, feel free to post a reply with any tweaks and I'll get it reflected within the spec.
